docs/sql-programming-guide.md

```diff
@@ -2430,6 +2430,7 @@ Spark SQL supports the vast majority of Hive features, such as:
 * Sampling
 * Explain
 * Partitioned tables including dynamic partition insertion
+  * Simply use the same syntax in Hive, after running `SET hive.exec.dynamic.partition.mode=nonstrict`
```

Comment:
This isn't quite a feature like the other elements in the list. Can it be a shorter elaboration on the previous point? or is this better documented elsewhere rather than in just the table of contents, so to speak?